src/redux/users-reducer.ts

```diff
@@ -17,18 +17,13 @@ const TOGGLE_IS_FOLLOWING_PROGRESS = 'TOGGLE_IS_FOLLOWING_PROGRESS';
 //   error: string
 // }
 
-
-type FollowingInProgressType = {
-  userId: number
-}
-
 let initialState = {
   users: [] as Array<UserType>,
-  pageSize: 20 as number | null,
-  totalUsersCount: 0 as number | null,
-  currentPage: 1 as number | null,
-  isFetching: true as boolean | null,
-  followingInProgress: [] as Array<FollowingInProgressType>
+  pageSize: 20, // as number | null или as number
+  totalUsersCount: 0, // as number | null,
+  currentPage: 1, // as number | null,
+  isFetching: true, // as boolean | null,
+  followingInProgress: [] as Array<number>
```

Comment:
remove unused code

Reply:
For educational purposes I will leave this code